corehq/apps/dashboard/static/dashboard/ng_partials/paginated_tile.html

```diff
@@ -14,17 +14,33 @@
                ng-show="showItemList()"></i>
             <i class="fa fa-spinner fa-spin fa-5x" ng-show="showSpinner()"></i>
             <div class="list-group" ng-show="showItemList()">
-                <a class="list-group-item"
-                   href="{{ item.url }}"
-                   data-ng-repeat="item in paginatedItems"
-                   popover-placement="right"
-                   popover-title="{{ item.name_full }}"
-                   popover="{{ item.description }}"
-                   popover-trigger="mouseenter"
-                   track-analytics=""
-                   title="">
-                    {{ item.name }}
-                </a>
+                {% if item.secondary_url and item.secondary_url_icon %}
```

Comment:
Be careful with this, as angular templates may share similar syntax with django templates but ultimately they are two different things. See the `ng-if` directive for what you'd like to do here: https://docs.angularjs.org/api/ng/directive/ngIf